### ISSUE TITLE
Include content blocks in website search broken for SS4 - waiting got fulltextsearch upgrade

### ISSUE THREAD

**Issue by pitchandtone** [dnadesign/silverstripe-elemental#88]
Currently architecture will be fragile once we add https://github.com/dnadesign/silverstripe-elemental/issues/84

**Comment by robbieaverill** [comment]
Progress for SS4 on fulltextsearch: https://github.com/silverstripe/silverstripe-fulltextsearch/pull/141

**Comment by robbieaverill** [comment]
cc @nicolewilliams16 - this is blocked by a fulltextsearch upgrade to SS4 at the moment

**Comment by robbieaverill** [comment]
Full text search is now SS4 compatible, and once https://github.com/silverstripe/silverstripe-fulltextsearch/pull/173 is merged should be testable with this module.

I've tested by indexing element data and returning it in a frontend search context, which is fine.

I've suggested that we add an indexer that will use combined element content for a page and index it in Solr as the page content, to ensure default frontend search queries will show matched content under the context of the page that owns the matched content.
